cvat-ui/src/components/app/app.tsx

```diff
@@ -35,12 +26,15 @@ class App extends PureComponent<any, any> {
     return(
       <Router>
         <div>
-          <Redirect from="/" to="dashboard" />
```

Comment:
Why did you cancel redirection?

Reply:
There are a couple of issues with this right now. I will return to this later to provide more robust approach for redirections.

Reply:
Could you please add TODO marker into the code?